bin/.editorconfig

```diff
@@ -0,0 +1,9 @@
+[*]
```

Comment:
This format works for all files here, and, hey, it turns out github supports this!